harvey/setplot.py

```diff
@@ -4,16 +4,17 @@
 
 import os
 
-import numpy
+import numpy as np
```

Comment:
The preferred style in GeoClaw is to leave the import of `numpy` as is (although we are sometimes inconsistent).